task_8/Anton.Panciuc/src/main/java/com/hillel/flashlight1/Flashlight.java

```diff
@@ -0,0 +1,76 @@
+package com.hillel.flashlight1;
+
+public class Flashlight {
+
+  public static final int MAXIMUX_BATTERIES = 3;
+  private String name;
+  private int batteries;
+  private boolean shines;
+  private String color;
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public int getBatteries() {
+    return batteries;
+  }
+
+  public boolean isShines() {
+    return shines;
+  }
+
+  public void insertBatteries() {
+    if (batteries == 3) {
+      System.out.println("Maximum capacity reached");
+    } else {
+      ++batteries;
+      if (batteries > 0) {
+        shines = true;
+      }
+
+    }
+  }
+
+  public String getColor() {
+    return color;
+  }
+
+  public void setColor(int color) {
+    switch (color) {
+      case 1:
```

Comment:
It's better to use constants here instead of just numbers. You can use enum construction here